CHANGELOG.md

```diff
@@ -86,6 +86,165 @@ if input key is empty, or input data contains empty key.
 * [\#8953](https://github.com/cosmos/cosmos-sdk/pull/8953) Add the `config` CLI subcommand back to the SDK, which saves client-side configuration in a `client.toml` file.
 
 
+## [v0.42.4](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.42.4) - 2021-04-08
```

Comment:
The versions of change log are out of order...

Reply:
Which one? Looks like it's correctly ordered: v0.42.5, v0.42.4, v0.42.3, etc.